openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -112,6 +108,18 @@ class RelativeHybridTopologyProtocolSettings(Settings):
     class Config:
         arbitrary_types_allowed = True
 
+    protocol_repeats: int = 3
+    """
+    Number of protocol repeats to run.  Default 3
```

Comment:
since there's always confusion over replicates/repeats etc, maybe define repeats here, something like "The number of completely independent repeats of the entire sampling process.  The mean of the repeats defines the final estimate of FE difference, while the variance between repeats is used as the uncertainty."

Reply:
Changed this!